plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -425,6 +425,15 @@ public synchronized Optional<ConnectorOutputMetadata> finishCreateTable(
         TableInfo info = tables.get(tableName);
         requireNonNull(info, "info is null");
 
+        if (!computedStatistics.isEmpty() && !info.properties().containsKey(TableInfo.DEFAULT_LIMIT_PROPERTY)) {
+            ComputedStatistics statistic = computedStatistics.stream().findFirst().orElseThrow();
+            long rowCount = (long) getValue(BIGINT, 0, statistic.getTableStatistics().get(TableStatisticType.ROW_COUNT));
+            info = info.withProperties(ImmutableMap.<String, Object>builder()
+                    .putAll(info.properties())
+                    .put(TableInfo.DEFAULT_LIMIT_PROPERTY, rowCount)
```

Comment:
nit: static import `DEFAULT_LIMIT_PROPERTY`

Reply:
This will conflict with the schema property.